docs/pages/installation.mdx

```diff
@@ -454,6 +454,12 @@ For Teleport Community Edition, check the
 [Downloads](https://goteleport.com/download/) page for the most up-to-date
 information.
 
+Teleport Enterprise (Managed) provides a Enterprise download page in the Web UI. Select the user name
```

Comment:
That's the same name we're using in the rest of this page. Def we have used the cloud ref in other pages.

Reply:
We removed the Vale linter that standardizes product naming, and I'm not sure we really have a standard anymore in the docs. On the Pricing page (https://goteleport.com/pricing/), the one available edition is Enterprise, and the "Deployment" section of the feature list includes "Self-hosted" and "Cloud" options.

I would change this to "Teleport Enterprise provides a download page in the Web UI".

Reply:
ONly the Cloud instance provides a download though. The Enterprise self-hosted does not. You have to go to the account dashboard.

Reply:
👍 I would be more explicit while describing a way of deploying Teleport Enterprise, rather than an edition name:

```
On cloud-hosted Teleport Enterprise accounts, you can visit a download page in the Web UI.
```

Reply:
Please see update